pyvista/themes.py

```diff
@@ -0,0 +1,2071 @@
+
+    @property
+    def number_of_peels(self) -> int:
+        """Return or set the number of peels in depth peeling.
+
+        Examples
+        --------
+        >>> import pyvista
+        >>> pyvista.global_theme.depth_peeling.number_of_peels = 1
+
+        """
+        return self._number_of_peels
+
+    @number_of_peels.setter
+    def number_of_peels(self, number_of_peels: int):
+        self._number_of_peels = int(number_of_peels)
+
+    @property
+    def occlusion_ratio(self) -> float:
+        """Return or set the occlusion ratio in depth peeling.
+
+        Examples
+        --------
+        >>> import pyvista
+        >>> pyvista.global_theme.depth_peeling.occlusion_ratio = 0.0
+
+        """
+        return self._occlusion_ratio
+
+    @occlusion_ratio.setter
+    def occlusion_ratio(self, occlusion_ratio: float):
+        self._occlusion_ratio = float(occlusion_ratio)
+
+    @property
+    def enabled(self) -> bool:
+        """Return or set if depth peeling is enabled.
+
+        Examples
+        --------
+        >>> import pyvista
+        >>> pyvista.global_theme.depth_peeling.enabled = True
+
+        """
+        return self._enabled
+
+    @enabled.setter
+    def enabled(self, enabled: bool):
+        self._enabled = bool(enabled)
+
+    def __repr__(self):
+        txt = ['']
+        parm = {
+            'Number': 'number_of_peels',
+            'Occulusion ratio': 'occlusion_ratio',
+            'Enabled': 'enabled',
+        }
+        for name, attr in parm.items():
+            setting = getattr(self, attr)
+            txt.append(f'    {name:<21}: {setting}')
+        return '\n'.join(txt)
+
+
+class _SilhouetteConfig(_ThemeConfig):
+    """PyVista silhouette configuration.
+
+    Examples
+    --------
+    Set global silhouette parameters
+
+    >>> import pyvista
+    >>> pyvista.global_theme.silhouette.color = 'grey'
+    >>> pyvista.global_theme.silhouette.line_width = 2
+    >>> pyvista.global_theme.silhouette.feature_angle = 20
+
+    """
+
+    def __init__(self):
+        self._color = parse_color('black')
+        self._line_width = 2
+        self._opacity = 1.0
+        self._feature_angle = None
+        self._decimate = 0.9
+
+    @property
+    def color(self) -> tuple:
+        """Return or set the silhouette color.
+
+        Examples
+        --------
+        >>> import pyvista
+        >>> pyvista.global_theme.silhouette.color = 'red'
+
+        """
+        return self._color
+
+    @color.setter
+    def color(self, color: Union[tuple, str]):
+        self._color = parse_color(color)
+
+    @property
+    def line_width(self) -> int:
+        """Return or set the silhouette line width.
+
+        Examples
+        --------
+        >>> import pyvista
+        >>> pyvista.global_theme.silhouette.line_width = 2
+
+        """
+        return self._line_width
+
+    @line_width.setter
+    def line_width(self, line_width: int):
+        self._line_width = int(line_width)
```

Comment:
Can't the line width be float?

Reply:
Correct:
https://vtk.org/doc/nightly/html/classvtkProperty.html#ab4f68c85e141af4c09e1827c94cbfa5b

Fixing...